string command
float64[] params
---
bool success
string message
